packages/www/components/HomeHero/index.tsx

```diff
@@ -200,9 +200,10 @@ const HomeHero = () => {
             <Text sx={{ fontSize: "18px", lineHeight: 1.7 }}>
               By leveraging decentralized infrastructure, Livepeer.com delivers
               a video platform that focuses on what user-generated content (UGC)
-              applications need. Affordable, scalable and reliable, the Livepeer
-              platform combines high-quality transcoding with powerful APIs and
-              features that are essential for a seamless creator experience.
+              applications need. Affordable, scalable and reliable, the
```

Comment:
who do i need to fight to get an oxford comma added here (ง •̀_•́)ง

Reply:
I gotchu. Oxford comma added in [04ff25](https://github.com/livepeer/livepeer-com/pull/25/commits/04ff2537a397629316e3ccef0ffba6a97665ad0a)